cloudomate/util/captchasolver.py

```diff
@@ -0,0 +1,153 @@
+import json, requests
+from time import sleep
+import os
+import base64
+
+"""
+use this already generated API key for testing: fd58e13e22604e820052b44611d61d6c
+
+Usage: 
+
+Feed Anti Captcha account API key to solver
+c_solver = captchaSolver("fd58e13e22604e820052b44611d61d6c")
+
+Find out how much money us left
+temp = c_solver.getBalance()
+
+Provide captcha image full path to solver method to get solution
+solution = c_solver.solveCaptchaTextCaseSensitive("/home/testm2/Desktop/testcscr/php/pyhton/cap2.jpg")
+print(str(solution))
+"""
+
+class captchaSolver():
```

Comment:
Make this a subclass of `object`. Also, class names should start with a capital letter (`CaptchaSolver`).